clap_builder/src/builder/possible_value.rs

```diff
@@ -146,6 +163,16 @@ impl PossibleValue {
         self.aliases.extend(names.into_iter().map(|a| a.into()));
         self
     }
+
+    /// Add multiple _visible_ aliases to this [PossibleValue].
+    ///
+    /// Unlike [PossibleValue::aliases], these aliases will show in help output.
+    #[must_use]
+    pub fn visible_aliases(mut self, names: impl IntoIterator<Item = impl Into<Str>>) -> Self {
```

Comment:
We'll need to add support for this to the dynamic completion system.

See #5585 and a follow up PR that will be linked to from #3951